docs/api.md

```diff
@@ -34,9 +34,11 @@ follow a structure similar to superagent's request API itself.
 the origin.
 - `redirects` *(Number)*: number of redirects to follow.
 - `eager` *(Boolean)*: whether or not to execute the request regardless of
-  usage of its corresponding response. Default value is `false` (i.e.,
-  the request is lazy). Main use case is: set this option to `true` if you
-  send POST requests and you are not interested in its response.
+  usage of its corresponding response. By default the eager setting of the
+  driver is used (whose default is `false`, i.e. the request is lazy). 
+  Explicitely setting eager in the request always overrides the driver 
+  setting. Main use case is: set this option to `true` if you send POST requests and
+  you are not interested in its response. 
```

Comment:
Can I ask you to move these changes out of this file and into src/index.js? This file is auto-generated from that, so we never should directly modify this one.